cmd/kpod/common.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	is "github.com/containers/image/storage"
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var needToShutdownStore = false
+
+func getStore(c *cli.Context) (storage.Store, error) {
+	options := storage.DefaultStoreOptions
+	if c.GlobalIsSet("root") || c.GlobalIsSet("runroot") {
```

Comment:
I don't know, that was in some code that Dan gave me when I started working on `kpod images` and `kpod rmi`.  I can tell you though, that the global flags aren't defined in this PR.  They are defined in the kpod images/rmi PR though.  I can add them to this one too if you'd like